tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/jsf/JSFTemplateNamePossibleValuesService.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ *******************************************************************************/
+
+package com.liferay.ide.project.core.modules.jsf;
+
+import java.util.Set;
+
+import org.eclipse.sapphire.PossibleValuesService;
+import org.eclipse.sapphire.Value;
+
+/**
+ * @author Simon Jiang
+ */
+public class JSFTemplateNamePossibleValuesService extends PossibleValuesService
+{
+
+    private String[] JSF_TEMPLATES = {"JSF Standard", "Liferay Faces Alloy", "ICEFaces", "PrimeFaces", "RichFaces"};
```

Comment:
No need for this to be a service, we should be able to add this directly as an annotation on the model class.